easybuild/easyconfigs/r/R-keras/R-keras-2.2.5.0-fosscuda-2019b-Python-3.7.4-R-3.6.2.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'Bundle'
+
+name = 'R-keras'
+version = '2.2.5.0'
+versionsuffix = '-Python-%(pyver)s-R-%(rver)s'
+
+homepage = 'https://cran.r-project.org/web/packages/keras'
+description = """ Interface to 'Keras' <https://keras.io>, a high-level neural networks 'API'. """
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('R', '3.6.2'),
+    ('Keras', '2.3.1', '-Python-%(pyver)s'),
+]
+
+exts_defaultclass = 'RPackage'
+exts_default_options = {
+    'source_urls': [
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'https://cran.r-project.org/src/contrib/',  # current version of packages
+        'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+exts_list = [
+    ('config', '0.3', {
+        'checksums': ['5ade6a768db6225634095ac0e71e5710fd1597c15512bac0b45a0b52afb3c750'],
+    }),
+    ('tfruns', '1.4', {
+        'checksums': ['6fc31f3b75c1888d3d14fe14a31c3567dab1bc2a4d0a61c0c72d543c422d1bd7'],
+    }),
+    ('tensorflow', '2.0.0', {
```

Comment:
Please use 2.1

Reply:
There isn't a cran package for 2.1.0 - but the existing cran 2.0.0 should work with a tf 2.1.0 backend, I hope...

Reply:
Oh so this is not actually the TF version used?

Reply:
A slight delay here, but no, the R extension named "TensorFlow" is a binding library that mostly tracks TensorFlow proper. Mostly.